ui/tests/integration/components/database-role-edit-test.js

```diff
@@ -97,11 +98,16 @@ module('Integration | Component | database-role-edit', function (hooks) {
     await render(hbs`<DatabaseRoleEdit @model={{this.modelStatic}} @mode="create"/>`);
     await fillIn('[data-test-ttl-value="Rotation period"]', '2');
     await click('[data-test-toggle-input="toggle-skip_import_rotation"]');
+    await fillIn('[data-test-input="password"]', 'testPassword'); // fill in password field
```

Comment:
can you use the [general selectors ](https://github.com/hashicorp/vault/blob/main/ui/tests/helpers/general-selectors.ts#L45)here? I know it's not used elsewhere, but moving forward, we are slowly (very slowly) trying to implement this pattern.